### PR TITLE
Fix code snippet in README

### PR DESCRIPTION
**Pull request by sunank200**
# Description
## What is the current behavior?

The code snippet is not being rendered in README. More context on https://astronomer.slack.com/archives/C03868KGF2Q/p1680181382198169

closes: #49 

## What is the new behavior?

- Add the commit sha in URL for code snippet

## Does this introduce a breaking change?
No

### Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by utkarsharma2 — APPROVED** [review]

**Comment by pankajkoti** [comment]
> can you please include a screenshot of how it looks like post this change?

There is a rich diff viewer that Github provides on PRs. Tatiana pointed me to this recently which could be helpful to take a quick look on how it looks after the change

Screenshot 2023-03-31 at 1 27 09 PM

File: README.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ Internally, the steps involved are:
 
 Following is an example of non-native transfers between Google cloud storage and Sqlite:
 
-https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_universal_transfer_operator.py#L37-L41
-
+https://github.com/astronomer/apache-airflow-provider-transfers/blob/a80dc84b7f33bb86ae244f79411b240f4f4c7e22/example_dags/example_universal_transfer_operator.py#L45-L54
```
Comment (pankajkoti) [inline README.md:50]: This example is for S3 to GCS but the above line says GCS and sqlite.

File: README.md [hunk 2]
```diff
@@ -65,24 +64,23 @@ Steps:
...
 
 ### Transfer using a third-party tool
-The Universal Transfer Operator can also offer an interface to generic third-party services that transfer data, similar to Fivetran.
+The `UniversalTransferOperator` can also offer an interface to generic third-party services that transfer data, similar to Fivetran.
 
 Here is an example of how to use Fivetran for transfers:
 
-https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_dag_fivetran.py#L52-L58
-
+https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_dag_fivetran.py#L55-L61
```
Comment (pankajkoti) [inline README.md:71]: Can we also change this URL to fetch it from a specific commit ID?

Comment (sunank200) [reply README.md]: [Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/50/commits/4b9427e8259f407873530389d2686f9b4597a10a)

File: README.md (hunk 1 above)
Comment (sunank200) [reply README.md]: Changed: https://github.com/astronomer/apache-airflow-provider-transfers/pull/50/commits/0def63336d40d45f28fbe4b11941805910906b69

**Pull request merged**